Data/Set.hs

```diff
@@ -152,15 +159,10 @@ module Data.Set (
             -- ** List
```

Comment:
The two list sections should probably be merged.

Reply:
Done.